xml/System.Runtime.InteropServices/RuntimeInformation.xml

```diff
@@ -184,7 +184,18 @@ The property returns a string that indicates the name of the currently executing
       <Docs>
         <summary>Gets the platform architecture on which the current app is running.</summary>
         <value>The platform architecture on which the current app is running.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This returned value is intended to represent the actual architecture of the underlying operating system. It is designed
+to ignore architecture emulation infrastructure that may be involved to run the process.
+
+On .NET 6, when running emulated as X86 on Windows ARM64, the returned value will be X64 not ARM64.
```

Comment:
This can be either X86 or X64, depending on Windows version that you are running.

I am not sure we need to document this. We have agreed that this was a bug in .NET 6 and earlier versions. We do not document bugs.

Reply:
@richlander Any pushback to this?

Reply:
On reflection, I agree.. I had stuck in my mind that it was this value on .NET 6 back to .NET Framework, which would be worth documenting, but it's not and I agree we don't generally document bugs.

Reply:
Thanks @danmoseley. I'll remove it. Should we ensure something goes in the release notes or is the issue sufficient?

Reply:
@richlander ?

Reply:
I think a breaking changes note is sufficient. No concerns from me.

Reply:
I added the label to trigger the BCN mechanism into https://github.com/dotnet/runtime/pull/60910